pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -78,3 +31,74 @@ func TestOryIntegration(t *testing.T) {
 		require.Equal(t, true, ready)
 	}
 }
+
+func TestOryIntegrationProduction(t *testing.T) {
+	skipTestIfDisabled(t)
+
+	if !isProductionProfile() {
+		t.Skipf("Integration tests disabled: skipping parts of test case '%s'", t.Name())
+	}
+
+	setup := newOryTest(t)
+	defer setup.contextCancel()
+
+	t.Run("ory-hydra on production profile is deployed", func(t *testing.T) {
+		name := "ory-hydra"
+		hpa := setup.getHpa(t, name)
+
+		require.GreaterOrEqual(t, int32(1), hpa.Status.CurrentReplicas)
```

Comment:
Should we check that current replicas is not greater than `hpa.Spec.MaxReplicas` or this would be unnecessary?

Reply:
that would mean that HPA already did the scale-up, so I guess it is unnecessary